build/tasks/generate-imports.js

```diff
@@ -0,0 +1,83 @@
+/*eslint-env node */
+const UglifyJS = require('uglify-js');
+
+module.exports = grunt => {
+	grunt.registerMultiTask(
+		'generate-imports',
+		'Task for generating an axe.imports module with external dependencies.',
+		function() {
+			// Convenience method that utilises uglifyjs tree-transformer to unwrap umd module resolver
+			const removeUMD = new UglifyJS.TreeTransformer(node => {
+				if (node.body[0].body.args.length <= 0) {
+					throw new Error('Not a UMD wrapper as arguments are missing.');
+				}
+
+				// the last (or only) argument in umd resolver is the factory to be mounted
+				const umdFactory =
+					node.body[0].body.args[node.body[0].body.args.length - 1];
+				const funcCall = new UglifyJS.AST_Call({
+					expression: umdFactory,
+					args: umdFactory.argnames // pass arguments into self invoking func
+				});
+				const statement = new UglifyJS.AST_SimpleStatement({
+					body: funcCall,
+					start: {
+						comments_before: node.start.comments_before // bring over comments
+					}
+				});
+				return new UglifyJS.AST_Toplevel({ body: [statement] });
+			});
+
+			// Convenience method that uses uglifyjs to parse a given code and run a transformer
+			const unwrappedCode = (sourceCode, cb) => {
+				try {
+					const unWrappedCode = UglifyJS.parse(sourceCode)
+						.transform(removeUMD)
+						.print_to_string({ comments: true });
+					cb(null, unWrappedCode);
+				} catch (e) {
+					cb(e, null);
+				}
+			};
+
+			const hasUmdWrapper = sourceCode => {
```

Comment:
This is a little scary and seems brittle. I think it's fine for now, since we *know* Axios has a UMD wrapper, but before we start adding more "imports", we may want to revisit this.